releases.md

```diff
@@ -42,7 +42,7 @@ Further documentation available:
 ## Release
 ### v0.56 (LTS)
 
-- **Latest Release**: [v0.56.0][v0.56-0] (2024-01-19) ([docs][v0.56-0-docs], [examples][v0.56-0-examples])
+- **Latest Release**: [v0.56.1][v0.56-1] (2024-02-13) ([docs][v0.56-1-docs], [examples][v0.56-1-examples])
```

Comment:
NIT: we might also need to update https://github.com/tektoncd/pipeline/pull/7663/files#diff-1d2b91d957f8612017f196e6cc2e83c33bcfe3fd348dc3619e9e9672e797d46fR190

i.e. following https://github.com/tektoncd/pipeline/commit/f990d8cb79762fe126681e71622ead8efb1f4ba8

Reply:
Agreed! Thanks for that! I submitted a new PR to fix this.